examples/00-README-samples.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Sends a HTTP request a returns its body as a Json array.
  */
-function getJsonResponseAsync(Tornado\HttpClient $httpClient, RequestInterface $request): \Generator
```

Comment:
Is this modification intended? Sounds better to me to let `\Generator` 🤷‍♂️

Reply:
🤔 This is from cs-fixer. Sounds weird to me too, I will check why does it did this

Reply:
I think I understand a bit better now. Only Generator in non-namespaced files are affected. I guess this is because there is no confusion possible?

Reply:
I updated the namespace for the tests, it fixed the problem :)